tests/fe/fe_rannacher_turek_01.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2015 by the deal.II authors
+// Copyright (C) 2015, 2017 by the deal.II authors
```

Comment:
In other words, thinking about these things is a waste of developer and reviewer time. I'll claim that it's an even bigger waste if we make people fix these things in their patches -- this is just what emacs does to my files, so so be it.